docs/changehistory/NextVersion.md

```diff
@@ -1490,6 +1490,22 @@ The property `InterpolationCurve3dOptions.isChordLenTangent` has been deprecated
 The iModel Transformer APIs, such as the classes [IModelExporter]($transformer), [IModelImporter]($transformer), and [IModelTransformer]($transformer)
 were removed from the `@itwin/core-backend` package and moved to a new package, `@itwin/core-transformer`.
 
+## transformer instance property settings are now deprecated in favor of initialization-only settings
```

Comment:
@pmconne
I interpreted your mention of NextVersion as referring to the the deprecations. I agree they belong here. Please let me know if I misinterpreted and you thought something else such as the dead predecessor option's description belongs here. If I didn't, please resolve

Reply:
Deprecations for sure, but also new features. I cleaned up the transformer-related change docs and left a TODO for you to fill in.

Reply:
please review the filled in section and changes to the other transformer description